src/app/pages/home-page/home-page.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="container">
+  <app-article
+    *ngFor="let article of articles"
+    [article]="article"
+    (increaseFavorite)="articleEvent($event)"
+  ></app-article>
```

Comment:
better to define article-list. This component will be a smart component, maintain all the data related to the articles. HomePage should not know anything about the article list inner behaviour, just present a view of list, no more